internal/operator-controller/rukpak/convert/helm_test.go

```diff
@@ -24,7 +24,10 @@ func Test_BundleToHelmChartConverter_ToHelmChart_ReturnsBundleSourceFailures(t *
 	var failingBundleSource FakeBundleSource = func() (bundle.RegistryV1, error) {
 		return bundle.RegistryV1{}, errors.New("some error")
 	}
-	_, err := converter.ToHelmChart(failingBundleSource, "install-namespace", "watch-namespace")
+	config := map[string]interface{}{}
+	config[bundle.BundleConfigInstallNamespaceKey] = "install-namespace"
+	config[bundle.BundleConfigWatchNamespaceKey] = "watch-namespace"
```

Comment:
nit:
```suggestion
	config := map[string]interface{}{
	    bundle.BundleConfigInstallNamespaceKey: "install-namespace",
	    bundle.BundleConfigWatchNamespaceKey: "watch-namespace",
	}
```

same for the others